### PR TITLE
Unify liquidity

### PR DESCRIPTION
**Pull request by jcompagni10**

### CODE REVIEW

File: proto/duality/dex/fee_tier.proto [hunk 1]
```diff
@@ -5,6 +5,6 @@ option go_package = "github.com/NicholasDotSol/duality/x/dex/types";
 
 message FeeTier {
   uint64 id = 1;
-  int64 fee = 2; 
-  
+  uint64 fee = 2;
```
Comment (teddyknox) [inline proto/duality/dex/fee_tier.proto:8]: 👍🏻

**Review by teddyknox — COMMENTED** [review]
Review pt 1

**Comment by jcompagni10** [comment]
Should be pretty much ready to go. Will continue to do some cleanup. But content-wise I think this is pretty solid. 

A couple things that I will continue to work on fixing:
-pool.save tries to delete non-existent ticks (ie. created via `GetOrInitTickLP` but never used). Not a real problem, but a bit inefficient 
-we don’t actually return errors from within the `LiquidityIterator` loop
-Some tests that relied on Min/Max tick need to be deleted or reworked because they now pointless 
-Figure out if tradingPairs are worth keeping and how we want to handle them from GRPC side

File: x/dex/keeper/liquidity.go [hunk 2]
```diff
@@ -1,206 +1,109 @@
...
+	}()
+
+	for ; s.iter.Valid(); s.iter.Next() {
+		tick := s.iter.Value()
+		switch tick.LiquidityType {
+		case types.LiquidityTypeLP:
+			var err error
+			var pool Liquidity
+			if s.is0To1 {
+				pool, err = s.createPool0To1(tick)
```
Comment (teddyknox) [inline x/dex/keeper/liquidity.go:54]: TODO: Remove?

**Review by teddyknox — DISMISSED** [review]
Going to stamp and we shall proceed forward with this review with a series of improvements on main.

**Review by teddyknox — APPROVED** [review]

**Pull request merged**